src/test/java/org/broadinstitute/hellbender/tools/walkers/variantutils/SelectVariantsIntegrationTest.java

```diff
@@ -0,0 +1,751 @@
+package org.broadinstitute.hellbender.tools.walkers.variantutils;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+import org.broadinstitute.hellbender.CommandLineProgramTest;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.IntegrationTestSpec;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collections;
+
+public class SelectVariantsIntegrationTest extends CommandLineProgramTest {
+
+    // For now we're retaining hardcoded paths to the test files on the broad server
+    // Any tests dependent on these files are in the testNG group "requiresRemoteAccess"
+    // To enable these tests (which require access to the  broad files), set remoteTestsEnabled=true
+    private static final boolean remoteTestsEnabled=false;
+
+    private static final String b36KGReference = "/humgen/1kg/reference/human_b36_both.fasta";
+    private static final String b37KGReference = "/humgen/1kg/reference/human_g1k_v37.fasta";
+    private static final String GATKDataLocation = "/humgen/gsa-hpprojects/GATK/data/";
```

Comment:
We need to completely remove the dependency on files in the Broad filesystem (and remove the `requiresRemoteAccess` group), even if it means changing the test data quite a bit. Could you perhaps refactor to use slices of `src/test/resources/large/dbsnp_138.b37.1.1-65M.vcf` as your input, and not use a reference at all in the tests (since it's optional in the hellbender version of the tool)?